iocage_cli/export.py

```diff
@@ -21,16 +22,75 @@
 # STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING
 # IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
-"""export module for the cli."""
+"""Export a jail from the CLI."""
 import click
+import os.path
 
-import iocage_lib.iocage as ioc
+import iocage.errors
+import iocage.Filter
+import iocage.Jail
+import iocage.Jails
+import iocage.Logger
+import iocage.Releases
+import iocage.Resource
+
+from .shared.click import IocageClickContext
 
 __rootcmd__ = True
 
 
-@click.command(name="export", help="Exports a specified jail.")
+@click.command(name="export", help="Export a jail to a backup archive")
+@click.pass_context
 @click.argument("jail", required=True)
-def cli(jail):
-    """Make a recursive snapshot of the jail and export to a file."""
-    ioc.IOCage(jail=jail).export()
+@click.argument("destination", required=True)
+@click.option(
+    "-s", "--standalone",
+    default=False,
+    is_flag=True,
+    help="Exports the jails root dataset independently"
+)
+# @click.option(
+#     "-r", "--recursive",
+#     default=False,
+#     is_flag=True,
+#     help="Include ZFS snapshots. Implies --standalone"
```

Comment:
why is this commented out?

Reply:
It is disabled because it currently cannot be imported but will be enabled in a later step. I don't want users to create backups they have to manually import later on.